src/routes/import/video.svelte

```diff
@@ -26,7 +32,7 @@
 		Sanipasse demande la permission d'accéder à votre webcam pour lire le QR code.
 	</p>
 	<p class="text-center">
-		Votre navigateur va vous demander si vous souhaitez autoriser <i>sanipasse.fr</i> à utiliser votre
+		Votre navigateur va vous demander si vous souhaitez autoriser <i>{domain}</i> à utiliser votre
```

Comment:
Peut-être que l'on peut le faire avec une simple condition ici ?

```suggestion
		Votre navigateur va vous demander si vous souhaitez autoriser <i>{globalThis?.location?.host || "sanipasse.fr"}</i> à utiliser votre
```

Reply:
ou pour la compatibilité 

```suggestion
		Votre navigateur va vous demander si vous souhaitez autoriser <i>{typeof location === "object" ? location.host : "sanipasse.fr"}</i> à utiliser votre
```

Reply:
Je ne connais pas du tout Svelte, d'instinct le `globalThis`ne me semble pas une bonne pratique.

On peut rajouter la compatibilité, je le ferais de cette manière

```js
domain = typeof window.location === "object" ? window.location.host : "sanipasse.fr";
```

Reply:
pourquoi est-ce que `globalThis` serait un problème, en dehors de l'incompatibilité avec les anciens navigateurs ? 

Vous ne pouvez pas faire `typeof window.location` côté serveur (sur node). Il faut faire `typeof window`.